manifests/s/Stoplight/Studio/2.10.0/Stoplight.Studio.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Stoplight.Studio
+PackageVersion: 2.10.0
+PackageLocale: en-US
+Publisher: Stoplight
+PublisherUrl: https://stoplight.io/
+PublisherSupportUrl: https://support.stoplight.io/
+PrivacyUrl: https://stoplight.io/privacy
+Author: Stoplight
+PackageName: Stoplight Studio
+PackageUrl: https://stoplight.io/studio
+License: CC BY-ND 4.0
+LicenseUrl: https://github.com/stoplightio/studio#license
+Copyright: Copyright © 2022 Stoplight
+ShortDescription: The modern editor for API Design and Technical Writing.
+Moniker: Stoplight Studio
+ReleaseNotesUrl: https://github.com/stoplightio/studio/releases/tag/v2.10.0-stable.8821.git-e47fcf5
+Documentations:
+- DocumentUrl: https://meta.stoplight.io/docs/platform/9f60f2fd436bd-api-design-overview

```

```suggestion
- DocumentUrl: https://meta.stoplight.io/docs/platform/9f60f2fd436bd-api-design-overview
  DocumentLabel: API Design Overview
```
Better to have a doc label as well.